Lib/distutils/sysconfig.py

```diff
@@ -449,7 +450,14 @@ def _init_nt():
     # XXX hmmm.. a normal install puts include files here
     g['INCLUDEPY'] = get_python_inc(plat_specific=0)
 
-    g['EXT_SUFFIX'] = _imp.extension_suffixes()[0]
+    # if cross-compiling replace hardcoded platform-specific EXT_SUFFIX
+    # with an EXT_SUFFIX that matches the target platform
+    if get_platform() == get_target_platform():
```

Comment:
If we change the definition of `get_platform` as per my post to python-dev, then to handle this I'd add a `get_host_platform` which is only used for this check.